.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Update the contribution guide accordingly with the steps needed to configure a CDK app to successfully connect to a debugger? I don't think it's trivial for someone new-ish to the code base.

https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#connecting-the-vs-code-debugger